.github/workflows/build.yml

```diff
@@ -60,8 +62,10 @@ jobs:
         if: matrix.send-coverage == true
         run: |
           opam install bisect_ppx
+          mkdir -p $BISECT_FILE
           opam exec -- dune runtest --instrument-with bisect_ppx --force
-          opam exec -- bisect-ppx-report send-to Coveralls
+          opam exec -- bisect-ppx-report send-to Coveralls --coverage-path /tmp/coverage
```

Comment:
Coveralls returns:

```
{"message":"Job #1940.1","url":"https://coveralls.io/jobs/139417004"}
```

Do you have an idea of if we could have this URL easily accessible ?

Reply:
https://coveralls.io/github/ocaml/odoc